axon_schema.go

```diff
@@ -23,20 +24,32 @@ var orphanSequences []string
 // for each table, maps column names to their postgres data types
 var columnTypes = (map[string]map[string]string{})
 
+func parseSemver(version string) (major, minor int, err error) {
+	reg, err := regexp.Compile(`^(\d+)\.(\d+)`)
+	if err != nil {
+		return
+	}
+	parsedVersion := reg.FindStringSubmatch(version)
+
+	major, err = strconv.Atoi(parsedVersion[1])
+	if err != nil {
+		return
+	}
+	minor, err = strconv.Atoi(parsedVersion[2])
+	if err != nil {
```

Comment:
technically this is redundant. i think a more explicit "return 0, 0, err" would be defensible for readability but if you're going for the compact form i'd just omit the redundant block unless this is good go style (i dont know go style)

Reply:
I think less redundant is better, addressed in 58ec0c5ebd9c08dc84877f6a123429f0a198194d